webapp/client/src/components/FormFieldYesNo.test.js

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { fireEvent } from "@testing-library/dom";
+import userEvent from "@testing-library/user-event";
+import FormFieldYesNo from "./FormFieldYesNo";
+
+describe("FormFieldYesNo", () => {
+  let props;
+  const onChangeHandler = jest.fn();
+
+  beforeEach(() => {
+    props = {
+      fieldName: "fooName",
+      fieldId: "fooId",
+      label: {
+        text: "foo",
+      },
+      errors: [],
+      onChangeHandler: onChangeHandler,
+    };
+    render(<FormFieldYesNo {...props} />);
+  });
+
+  describe("When Ja clicked", () => {
+    beforeEach(() => {
+      fireEvent.click(screen.getByText("Ja"));
```

Comment:
What's your reasoning for using `fireEvent` over `userEvent` here? I don't fully understand the difference, but the docs for `fireEvent` say:

> Most projects have a few use cases for fireEvent, but the majority of the time you should probably use @testing-library/user-event.

Reply:
Good point. I started with fireEvent before using the userEvent library and just kept it. I have changed this.